### ISSUE TITLE
Table feature is there or not

### ISSUE THREAD

**Issue by nafisahmedkhan9** [zuck/jsdocx#1]

**Comment by zuck** [comment]
Hi @nafisahmedkhan9. Table support is not implemented yet but I'm open to contributions!

The basics of the output syntax are:

```
<w:tbl>
 <w:tblPr>
   <w:tblW w:w="5000" w:type="pct" />
 </w:tblPr>
 <w:tblGrid><w:gridCol/><w:gridCol/></w:tblGrid>
 <w:tr>
   <w:tc><w:p><w:r><w:t>left</w:t></w:r></w:p></w:tc>
   <w:tc><w:p><w:r><w:t>right</w:t></w:r></w:p></w:tc>
 </w:tr>
</w:tbl>
```

(credits to: https://www.toptal.com/xml/an-informal-introduction-to-docx)

**Comment by zuck** [comment]
An initial draft has been published with v0.1.10
